code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cpp

```diff
@@ -0,0 +1,24 @@
+// Part of cosmos from opengenus foundation
+
+#include <iostream>
+#include <vector>
+#include <algorithm>
+using namespace std;
+
+int main() {
+    int n;
+    vector<int> arr;
+    cout<<"enter number of numbers : ";
+    cin>>n;
+    for(int i = 0 ; i < n ; i++) {
+        int a;
+        cin>>a;
+        arr.push_back(a);
+    }
+
+    sort(arr.begin(), arr.end());
+
+    cout<<"biggest number : "<<arr[n-1];
+
+    return 0;
+}   
```

Comment:
You should probably add a newline at the EOF (By unix definition).

Reply:
you mean 25th line ?

Reply:
Yup